client/post-editor/editor-featured-image/index.jsx

```diff
@@ -26,12 +27,14 @@ import { recordTracksEvent } from 'state/analytics/actions';
 
 class EditorFeaturedImage extends Component {
 	static propTypes = {
-		maxWidth: React.PropTypes.number,
-		site: React.PropTypes.object,
-		post: React.PropTypes.object,
-		selecting: React.PropTypes.bool,
-		onImageSelected: React.PropTypes.func,
-		featuredImage: React.PropTypes.object,
+		maxWidth: PropTypes.number,
+		site: PropTypes.object,
+		post: PropTypes.object,
+		selecting: PropTypes.bool,
+		onImageSelected: PropTypes.func,
+		featuredImage: PropTypes.object,
+		hasDropZone: PropTypes.bool,
+		isDropZoneVisible: PropTypes.bool,
```

Comment:
We are missing here the `recordTrackEvents` and `translate` props (the later is being added by the localize Higher Order Component). Also, are we using `hasDropZone` and `isDropZoneVisible`?